packages/all/package.json

```diff
@@ -1,31 +1,31 @@
 {
   "name": "@govuk-frontend/all",
-  "version": "0.0.19-alpha",
+  "version": "0.0.20-alpha",
```

Comment:
Is this normally done by lerna? Do we need to do this manually?

Reply:
👍  After consulting Lerna documentation it appears it's not necessary to change package.json manually. Have taken this out.